### ISSUE TITLE
do longer suggest pasting from clipboard

### ISSUE THREAD

**Issue by georg-d** [AnySoftKeyboard/AnySoftKeyboard#2842]
### Steps to reproduce
1. select some text and copy to clipboard
2. go to an text field and tap it - observe pasting is possible (top right, i.e. above keys)
3. press space - observe pasting vanishes so the bar is empty (i.e. space is not required for something else). Other variations see below.

### Expected behaviour
Keep showing paste icon (https://github.com/AnySoftKeyboard/AnySoftKeyboard/issues/1917) until first word suggestion becomes possible, not until ASK was used once. Motivation: Before pasting, I often need to 
- add a space, colon or the like 
- minimize ASK, scroll a little to look something up, and tap again the text field
- need to move cursor position a little

**Android OS version:**
10

**Device manufacturer and model:**
Fairphone 3

**Comment by PrSunflower** [comment]
Well described, I agree 100% with this.

In 1917 I suggested [doing like Windows 10 Mobile](https://github.com/AnySoftKeyboard/AnySoftKeyboard/issues/1917#issuecomment-571284433), a small clipboard icon is displayed indefinitely.

**Comment by voidplayer** [comment]
In my case is enter. I want to paste in a new line, and after pressing enter, the paste suggestion vanishes!

**Comment by georg-d** [comment]
@menny I tried in v1.12, past icon stays visible in all tries I did so far, but sometimes pressing it has no effect – in Markor is usually works (I can't yet reproduce the non-working variations), but in Microsoft OneNote tapping the icon does mostly (or always?) not paste; maybe that is related to https://github.com/AnySoftKeyboard/AnySoftKeyboard/issues/3531 (long pressing clipboard icon and pasting newsest/topmost entry does not work while the older/lower ones work fine).
IMHO this is a separate issue, hence, we shall keep #2842 closed.
